server/fixity.go

```diff
@@ -18,6 +32,8 @@ type FixityDB interface {
 	// the empty string is returned. It will also not return anything in an
 	// "error" state.
 	NextFixity(cutoff time.Time) string
+	GetFixityById(id string) *fixity
```

Comment:
Add comments describing these functions like the other ones in the interface.

Reply:
Done.